src/api/vipps-login/index.ts

```diff
@@ -61,12 +63,25 @@ export default (server: Hapi.Server) => () => {
         {nonce: nonce, state: state},
       );
       if (params.access_token) {
+        const vippsTimer = new Timer();
         const userInfo = await client.userinfo(params);
         const phoneNumber = userInfo.phone_number;
+        logResponse({
+          message: 'openid call',
+          requestHeaders: h.request,
+          duration: vippsTimer.getElapsedMs(),
+        });
         if (phoneNumber) {
           const auth = app.auth();
           try {
+            const authTimer = new Timer();
             const user = await auth.getUserByPhoneNumber(`+${phoneNumber}`);
+            logResponse({
+              message: 'firebase call',
+              requestHeaders: h.request,
+              duration: authTimer.getElapsedMs(),
+              customerAccountId: user.uid,
```

Comment:
Not sure about this, since I were unable to test it. Is `customerAccountId` = `uid` here?

Reply:
Not exactly sure, but if it's the Firebase Auth id, then no. It should be of the format `<codespace>:CustomerAccount:<id>`, e.g. for AtB we have `ATB:CustomerAccount:<id>`.

Reply:
We'll see in staging, then 🤞